core/src/graph/graph.rs

```diff
@@ -57,9 +57,10 @@ impl Graph {
 	}
 
 	/// Import bundle of pages as a Public Graph
-	pub fn import_public<E: EncryptionBehavior>(
```

Comment:
It seems like we might need something like a Multihash that specifies the key type. Or we might need the encryption type from spec config.